rails_async_migrations.gemspec

```diff
@@ -19,7 +19,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 2.1.4'
```

Comment:
I guess if the first one is safe, this one too :p

Reply:
I'll revert the changes to the gemfile and gem spec in this PR